docs/pages/inboxes/disappearing-messages.md

```diff
@@ -0,0 +1,97 @@
+# Support disappearing messages with XMTP
+
+Disappearing messages are messages that are intended to be visible to users for only a short period of time. After the message expiration time passes, the messages are removed from the UI and deleted from local storage so the messages are no longer accessible to conversation participants.
```

Comment:
> the messages are removed from the UI

I would say we can't guarantee this, we delete the message from the local db, but if the UI doesn't refresh on-time the message can be visible till the next fresh!

Reply:
Oh I see in the next paragraph you've mentioned that is the app's responsibility! ty🙌